plugins/lookup/merge_variables.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Thales Netherlands
+# Copyright (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    author:
+      - Roy Lenferink (@rlenferink)
+      - Mark Ettema (@m-a-r-k-e)
```

Comment:
Who is @m-a-r-k-e and how were they involved in this? Being added here and in the maintainers file means that they will get pinged for every issue / PR filed for this plugin. Is that OK?

Reply:
@rlenferink and I developed this plugin together. I was already following this, so it is OK to get pinged for any future issues/questions.

Reply:
Thanks for confirming that here. I don't like adding users who weren't involved (and where it is unclear whether they consent to it) :)